### DISCUSSION TITLE
Cubit does not recognize the svalinn/DAGMC plugin

### DISCUSSION THREAD

**Discussion by LukeLabrie** [svalinn/DAGMC#866] [Q&A] [answered]
I've installed cubit, as well as the cubit plugin for dagmc, and unpacked it into `/opt/Coreform-Cubit-2023.3`. I can see it in what I assume is the correct location. Output of `ls /opt/Coreform-Cubit-2023.3/bin/plugins`:
```
assetimporters         geoservices               platforms        renderplugins   virtualkeyboard
audio                  iconengines               platformthemes   sceneparsers    wayland-decoration-client
canbus                 imageformats              playlistformats  sensorgestures  wayland-graphics-integration-client
egldeviceintegrations  libmaterial_commands.ccm  position         sensors         wayland-graphics-integration-server
gamepads               libsvalinn_plugin.so      printsupport     sqldrivers      wayland-shell-integration
generic                mediaservice              qmltooling       svalinn         webview
geometryloaders        platforminputcontexts     renderers        texttospeech    xcbglintegrations

```
In cubit however, the command `help dagmc` is not recognized, and `export dagmc` is not functional. I've tried going into Tools -> Plugins and adding the path to the plugins folder, but this does not help. Although, when I open the plugins window, this is what I see:

![Screenshot_20230314_164545](https://user-images.githubusercontent.com/89009042/225057146-e4ac2ddf-217e-46ef-b35a-ef76ad597780.png)

Which seems like a lot fewer than are listed in the plugins folder...

Any ideas?

**Comment by LukeLabrie** [comment, marked as answer]
Fixed this by reverting to Cubit-2021 instead of Cubit-2023

**Reply by gonuke** [reply]
Great!  Now we need to update for 2023.3 as well...
